msticpy/vis/figure_dimension.py

```diff
@@ -0,0 +1,20 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+from bokeh.plotting import figure
+
+class figure_dimension:
```

Comment:
Actually, on second pass, does this need to be class?
Should just be a function, right?